templates/haproxy.cfg.j2

```diff
@@ -62,6 +62,8 @@ backend srv_{{ host }}
     option tcp-check
     tcp-check connect{{ " ssl" if o["health-check"] == "ssl" }}
         {% endif %}
-    server srv {{ o["hosts"][k] }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
+        {% for c in o["hosts"][k] %}
+    server {{ c }} {{ c }} check weight 1{{ " verify none" if o["health-check"] == "ssl" }}
```

Comment:
Could you replace `server {{ c }} {{ c }} ...` by `server srv{{ incr }} {{ c }} ...`

result output e.g.:

```
server srv1 test_agent:9001 check weight 1 verify none
```

https://cbonte.github.io/haproxy-dconv/2.4/configuration.html#5.2

Reply:
updated.

```
    server srv-0 f6d8d45b7411:80 check weight 1
    server srv-1 59b213cb8592:80 check weight 1

```